src/routes/user/index.ts

```diff
@@ -3,7 +3,6 @@ import * as deleteRoute from './delete';
 import * as find from './find';
 import * as get from './get';
 import * as update from './update';
-
```

Comment:
compared to other route index files, this deletion does not make sense.

Reply:
Updated